scrape.py

```diff
@@ -42,6 +42,20 @@
 
 # All the files we want to list
 files = [
+    # B2GDroid
```

Comment:
Please add these _after_ at least the Desktop builds, as I think they should be at the top of the Nightly page.